docker/Dockerfile

```diff
@@ -16,7 +16,6 @@ RUN chmod +x ./install.sh && \
 WORKDIR /root/sources
 RUN git clone https://github.com/sendgrid/sendgrid-java.git
 WORKDIR /root
-RUN ln -s /root/sources/sendgrid-java
```

Comment:
Why do you want to remove this line?

Reply:
An error occurs because there is no destination for the "ln" command.
What do you think?

Reply:
oops, sorry.
I did the wrong analysis.
This using 2rd form.
So, I close PR.